docs/deploy-monitoring-opentelemetry.md

```diff
@@ -0,0 +1,82 @@
+---
```

Comment:
<img width="337" alt="image" src="https://github.com/apache/pulsar-site/assets/989425/eb728cb5-ed83-4b35-9508-45ed8af2f35b">

Maybe this should be placed inside Monitoring?

Reply:
then from reference of `OpenTelemetry Monitoring` link to OpenTelemetry section in Monitoring?